browser/js/product/product.controller.js

```diff
@@ -0,0 +1,55 @@
+app.controller('ProductController', function ($scope, $state, product, ReviewFactory, Session, ProductsFactory, $location, $anchorScroll) {
+	$scope.product = product;
+	$scope.item = {quantity:1};
+	$scope.isAdmin = true;
+	// this should equal if the user is actuall an admin
```

Comment:
check out line 66 in browser/js/fsa/fsa-prebuilt. If you bring in the AuthService factory you can use the function 'isAdmin'!